Library/Homebrew/cmd/man.rb

```diff
@@ -51,7 +51,7 @@ def build_man_page
           map { |line| line.slice(2..-1) }.
           join
       }.
-      reject { |s| s.strip.empty? }
+      reject { |s| s.strip.empty? || s.include?("@hide_from_man_page") }
```

Comment:
> Maybe even make the matching a bit stricter and only accept the token if it is the only thing in a documentation line.

This chunk in one of my other code comment basically references this piece of code.